activejob/lib/active_job/core.rb

```diff
@@ -80,6 +80,7 @@ def serialize
       {
         "job_class"  => self.class.name,
         "job_id"     => job_id,
+        "provider_job_id" => provider_job_id,
```

Comment:
If the enqueue system does not provide a provider_job_id, will this still work? (I assume it will, but might be worth checking)